tools/mapping/gen_mapping_files/runoff_to_ocn/runoff_map.nml

```diff
@@ -0,0 +1,15 @@
+  &input_nml
+   gridtype     = 'rtm'
+   file_roff    = '/glade/p/cesm/cseg/inputdata/lnd/clm2/rtmdata/rdirc.05.061026'
+   file_ocn     = '/glade/p/cesm/cseg/mapping/grids/gx3v7_120309.nc'
+   file_nn      = 'map_r05_to_gx3v7_nn_130507.nc '
+   file_smooth  = 'map_gx3v7_to_gx3v7_sm_e1000r300_130507.nc '
+   file_new     = 'map_r05_to_gx3v7_nnsm_e1000r300_130507.nc'
+   title        = 'runoff map: r05 -> gx3v7, nearest neighbor and smoothed '
+   eFold        = 1000000.0
+   rMax         =  300000.0
+   step1 = .true.
```

Comment:
"glade" is an NCAR thing so ACME didn't add this.  Is something that was deleted on ESMCI and this merge is trying to add it back?